WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -8719,6 +8722,7 @@
 				98B11B8B2216536C00B7F2D7 /* StatsChildRowsView.xib in Resources */,
 				2FAE970C0E33B21600CA8540 /* xhtml1-transitional.dtd in Resources */,
 				4058F41A1FF40EE1000D5559 /* PluginDetailViewHeaderCell.xib in Resources */,
+				406A0EF0224D39C50016AD6A /* Flags.xcassets in Resources */,
```

Comment:
Should this diff also be accompanied by 258 deletions, reflecting removal of the standalone PNGs from `WordPressComStatsiOS`?

Reply:
Not yet, since we're gonna ship the new stats in the next release and those are required to remain in place for the "old" Stats to continue to work.